sunpy/net/jsoc/tests/test_jsoc.py

```diff
@@ -318,3 +330,17 @@ def test_results_filenames():
     assert len(files) == len(responses)
     for hmiurl in files:
         assert os.path.isfile(hmiurl)
+
+
+@pytest.mark.remote_data
+def test_results_filenames_as_is():
+    responses = client.search(
+        a.Time('2014/1/1T1:00:36', '2014/1/1T01:01:38'),
```

Comment:
How many files is this downloading? Also I can't see a significant difference between this and the other new test above, what are they testing?

In general tests which download data are slow and prone to failure, if we can adequately test this with a single download then we should.

Reply:
The default protocol is `fits`. In the newly added tests, the protocol is explicitly specified as `as-is` so as to check if the edits work correctly or not

Reply:
What I was trying to ask was you have two tests which both use `as-is` and to me they seem very similar, what's the difference between them?

Reply:
The first one has a single result, while the second one has 2.
Yeah, I believe we can remove the former test